packages/gatsby-theme-docs/src/layouts/internals/layout-page-navigation.js

```diff
@@ -289,6 +292,7 @@ const LayoutPageNavigation = (props) => {
 LayoutPageNavigation.displayName = 'LayoutPageNavigation';
 LayoutPageNavigation.propTypes = {
   isSearchBoxInView: PropTypes.bool.isRequired,
+  isReleaseNotesPage: PropTypes.bool,
```

Comment:
Could you add a brief comment about what this prop is doing, for future reference